crates/biome_module_graph/tests/snapshots/test_export_default_function_declaration.snap

```diff
@@ -1,7 +1,6 @@
 ---
 source: crates/biome_module_graph/tests/snap/mod.rs
 expression: content
-snapshot_kind: text
```

Comment:
What version of `cargo-insta` are you using? We may be different ones, but we should align. I use `cargo-insta 1.41.1`

Reply:
I was indeed at an old one (1.35.1), just bumped to 1.43.0 now.